src/ui/reducers/network.ts

```diff
@@ -105,10 +107,20 @@ export const getFocusedEvents = (state: UIState) => {
   return events.filter(e => e.time > beginTime && e.time <= endTime);
 };
 export const getFocusedRequests = (state: UIState) => {
```

Comment:
:hand: Hmm. Not sure what this was doing before, but this looks like it probably needs to be a memoized selector - right now it seems to be returning new array references every time, either from filtering or the empty array

Reply:
Agreed. Let's memoize that (and `getFocusedEvents`) in a follow up commit though since it's not new behaiovr?

Reply:
Nevermind. I'll just do it while I'm in there. Nbd.